documentation/modules/ROOT/pages/configuration/applying-transformations-selectively.adoc

```diff
@@ -0,0 +1,143 @@
+// Category: debezium-using
+// Type: assembly
+// ModuleId: applying-transformations-selectively-with-smt-predicates
+// Title: Applying transformations selectively with SMT predicates
+= Applying transformations selectively
+
+You can define a predicate for a single message transformation to specify a condition that event records must match before Kafka Connect applies the transformation.
```

Comment:
The predicate decides whether the SMT is applied _at all_, so I think this needs rewording.